tcb/.vscode/settings.json

```diff
@@ -16,7 +16,7 @@
   "torizon_app_root": "/home/torizon",
   "docker_tag": "v0.0.0",
   "tcb.packageName": "__change__",
-  "tcb.version": "3.11.0",
+  "tcb.version": "early-access",
```

Comment:
I think we must use the `3.12.0` here, I confirmed with Jeremias that this version works on TOS 7.

Reply:
Sure, updated